inngest/src/main/kotlin/com/inngest/Comm.kt

```diff
@@ -174,10 +174,10 @@ class CommHandler(
     private fun getRegistrationRequestPayload(origin: String): RegistrationRequestPayload =
         RegistrationRequestPayload(
             appName = config.appId(),
-            framework = framework.toString(),
-            sdk = "inngest-kt",
```

Comment:
interesting, I see the linear thread where you got `java` from. I feel like `JVM` or `JRE` is used in these situations but I can't think of a concrete example we can draw from

Reply:
yeah the runtime could be good too, same I know Sentry has both languages but the platform isn't auto-detected on there.